repose-aggregator/core/repose-core-api/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
 
     provided "org.glassfish:javax.servlet"
 
+    testCompile project(":repose-aggregator:core:repose-core")
```

Comment:
At first I was like, oh no! Cyclic dependency!

Then I was like, oh, nevermind, it's a `testCompile` dependency.

Getting the Repose version really is more of a pain than I would like... but I'll leave that for another time.